connector_prestashop/account.py

```diff
@@ -220,50 +216,40 @@ def _invoice_line(self, record, fpos):
         }
 
```

Comment:
in `_invoice_line`, `property_account_income_categ` has been renamed to `property_account_income_categ_id`